lib/travis/build/script/r.rb

```diff
@@ -126,9 +126,11 @@ def configure
                 elsif r_version == '3.2.5'
                   r_url = "#{repos[:CRAN]}/bin/macosx/old/R-3.2.4-revised.pkg"
 
-                # all other binaries are in /bin/macosx/old
-                else
+                # the old archive has moved after 3.4.0
+                elsif r_version < '3.4.0'
```

Comment:
You'd want to update this condition, too.

Reply:
OK will do right now